mimic/rest/identity_api.py

```diff
@@ -250,6 +250,43 @@ def get_users_details(self, request):
             "updated": seconds_to_timestamp(time.time())
         }))
 
+    @app.route('/v2.0/users/<string:user_id>/OS-KSADM/credentials',
+               methods=['GET'])
+    def get_user_credentials_osksadm(self, request, user_id):
+        """
+        Support, such as it is, for the credentials call.
+
+        reference: http://developer.openstack.org/api-ref-identity-v2-ext.html
+            #listCredentials
+        """
+        if user_id in self.core.sessions._userid_to_session:
+            username = self.core.sessions._userid_to_session[user_id].username
+            password = '9d5ed678fe57bcca610140957afab571'  # echo -n B | md5sum
+            apikey = '0d61f8370cad1d412f80b84d143e1257'  # echo -n C | md5sum
+            return json.dumps(
+                {
+                    'credentials': [
+                        {
+                            'passwordCredentials': {
```

Comment:
@glyph I was primarily following the API docs which I think we should be accurate to. If `mimic` had a way of storing that for the user, then it would return what was used instead of some random string.

That said, I'm not sure off hand. I know Cloud Backup does some stuff where we have to actually authenticate as the user, but I'm not sure off hand if we use password or apikey when doing so. When they asked everyone to use impersonation we ended up knocking down Identity; so this bit is pretty important. I know v2 is using apikey; I'll have to check regarding v1.